src/ethereum/arrow_glacier/fork.py

```diff
@@ -533,8 +533,8 @@ def apply_body(
 
     Returns
     -------
-    gas_available : `ethereum.base_types.Uint`
-        Remaining gas after all transactions have been executed.
+    gas_used : `ethereum.base_types.Uint`
```

Comment:
Maybe

```suggestion
    block_gas_used : `ethereum.base_types.Uint`
```

to match the actual variable name?

---

If you don't want to change it, I'm fine merging as-is too. We're switching to markdown shortly anyway.